opentelemetry-exporters-newrelic/src/test/java/com/newrelic/telemetry/opentelemetry/examples/BasicExample.java

```diff
@@ -106,15 +102,34 @@ public static void main(String[] args) throws InterruptedException {
 
     // clean up so the JVM can exit. Note: these will flush any data to the exporter
     // before they exit.
-    spanProcessor.shutdown();
     intervalMetricReader.shutdown();
+    spanProcessor.shutdown();
+  }
+
+  private static TelemetryClient createTelemetryClient(String apiKey) {
+    MetricBatchSender metricBatchSender =
+        MetricBatchSender.create(
+            MetricBatchSenderFactory.fromHttpImplementation(OkHttpPoster::new)
+                .configureWith(apiKey)
+                .auditLoggingEnabled(true)
+                .build());
+    SpanBatchSender spanBatchSender =
+        SpanBatchSender.create(
+            SpanBatchSenderFactory.fromHttpImplementation(OkHttpPoster::new)
+                .configureWith(apiKey)
+                .auditLoggingEnabled(true)
+                .build());
+
+    return new TelemetryClient(metricBatchSender, spanBatchSender, null, null);
+    // note: if you just want the simple defaults, you can use this method:
+    // return TelemetryClient.create(OkHttpPoster::new, apiKey);
```

Comment:
👍

Reply:
I'm wondering if this`BasicExample` shouldn't just use this simple way and then could be paired with a `FullyConfiguredExample` that shows how all the knobs may be adjusted.

Reply:
yeah, I went back and forth on that as well. I really like having audit logging enabled for the example, though, since you can see that it's working, even without looking at the NR UI.